src/playback/view/mixerpanelmodel.cpp

```diff
@@ -111,9 +83,11 @@ void MixerPanelModel::removeItem(const audio::TrackId trackId)
 {
     beginResetModel();
 
-    (void)std::remove_if(m_mixerChannelList.begin(), m_mixerChannelList.end(), [&trackId](const MixerChannelItem* item) {
-        return item->id() == trackId;
-    });
+    for (int i = m_mixerChannelList.count() - 1; i >= 0; i--) {
+        if (m_mixerChannelList[i]->id() == trackId) {
+            m_mixerChannelList.removeAt(i);
+        }
+    }
```

Comment:
Any reason not to use `remove_if / m_mixelChannel.erase( )`? (it works fine as far as I can tell). But this is definitely better than just using remove_if without erase.
Do we need to worry about deleting the removed elements though? Qt will clean them up when the MixelPanelModel is deleted, but I that only happens when you shut down, so there could still be a memory leak each time you change scores here.